environment.yml

```diff
@@ -2,7 +2,7 @@ name: toolbox
 channels:
   - conda-forge
 dependencies:
-  - python=3.11
+  - python=3.12
   - pip
   - matplotlib
   - numpy
```

Comment:
Do we also want to move to `numpy 2` in the same step?

Reply:
NumPy is already automatically installed as the latest version (2.1.1)

Reply:
I'm aware of that. That's why I'm asking - do we want to leave it open?

Reply:
I mean, we can set it to `>=2.0` or `~=2.0`... I wouldn't mind :+1:

Reply:
If everything works, I'm fine with it. Just recently had the experience with `numpy 2` to be lacking some important "legacy" functions depending on the project.
But if everything we need is also supported, then I'm fine with bumping the version.

Reply:
Notebook seems to run fine with NumPy 2